test/preview/conftest.py

```diff
@@ -11,3 +11,12 @@ def mock_tokenizer():
     tokenizer.encode = lambda text: text.split()
     tokenizer.decode = lambda tokens: " ".join(tokens)
     return tokenizer
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Can we maybe not put this on autouse? Also we're not even using it anymore or am I wrong? 🤔

Reply:
I would want `tenacity` to wait if it's actually running to be fair. That means a tests is making actual requests and it might be throlled if it doesn't wait.

Reply:
But we don't want tests to make requests, right? :eyes: This does not affect end to end tests.